test/test_rosbag/test/record_cleanup.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# Software License Agreement (BSD License)
+#
+# Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
It is fine to list Amazon for the copyright here. No need to credit Willow Garage here.

Reply:
Done